<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="fme_card_content_0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>245</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QFrame {
	border : 1px solid;
	border-color: #CED7E6;
    border-top-right-radius: 7px;
	border-top-left-radius: 7px;
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fme_image_cap_2">
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color : #CED7E6;
	border : 1px solid;
	border-color: #CED7E6;
    border-top-right-radius: 7px;
	border-top-left-radius: 7px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_19" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="fme_image">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fme_card_content_2">
        <property name="styleSheet">
         <string notr="true">QFrame {
	border : 1px solid;
	border-color: #CED7E6;
    border-top-right-radius: 0px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
	border : none;
	border-color: transparent;
    border-top-right-radius: 0px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_21" stretch="0,0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lbl_card_title">
              <property name="styleSheet">
               <string notr="true">QLabel {
	margin-top : 12px;
	font : 18px &quot;Arial&quot;;
	color: #34373B;
}</string>
              </property>
              <property name="text">
               <string>Category name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_card_subtitle">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font : 18px &quot;Arial&quot;;
	color: #A0A0A0;
}</string>
              </property>
              <property name="text">
               <string>13 art√≠culos</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
	border : none;
	border-color: transparent;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_22" stretch="0">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_10">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btn_show_items">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Mostrar productos</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	background-color : #DBE9FF;
	color : #6BA6FD;
	font : 17px &quot;Arial&quot;;
}

QPushButton:pressed {
	background-color : transparent;
	color : #6BA6FD;
	border: 1px solid #6BA6FD;
	border-radius: 5px;
}</string>
                 </property>
                 <property name="text">
                  <string>ver productos</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
